src/blocks/buttons-child/render.js

```diff
@@ -44,6 +44,8 @@ const Render = ( props ) => {
 		}
 		return null;
 	};
+	const isPro = uagb_blocks_info.spectra_pro_status;
+	const btnAllowedFormats = isPro ? ['uagb/dynamic-content'] : [];
```

Comment:
Since we're only using spectra_pro_status once, should we do this instead?

```suggestion
	const btnAllowedFormats = uagb_blocks_info.spectra_pro_status ? ['uagb/dynamic-content'] : [];
```